deploy/docker-compose/docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
       - 6703:6703
     depends_on:
       - caduceus
-      - tr1d1um
+      - goaws
```

Comment:
You can actually leave it like it was. After testing this way, I found that adding the delay in Hecate was enough.